### PR TITLE
Implement clusterguid config for etcd

### PR DESCRIPTION
**Pull request by mikev**

### CODE REVIEW

File: calico_containers/pycalico/datastore.py [hunk 1]
```diff
@@ -291,9 +292,31 @@ def ensure_global_config(self):
...
+            self.etcd_client.read(key)
+        except EtcdKeyNotFound:
+            guid = uuid.uuid4()
+            guid_string = guid.get_hex()
+            try:
+                self.etcd_client.write(key, guid_string, prevExist=False)
+            except EtcdAlreadyExist:
+                # ignore
+                pass
+            else:
```
Comment (fasaxc) [inline calico_containers/pycalico/datastore.py]: No need for this else block

**Comment by fasaxc** [comment]
Just one minor comment, otherwise looking good to me

File: calico_containers/pycalico/datastore.py (hunk 1 above)
Comment (mikev) [reply calico_containers/pycalico/datastore.py]: fixed.

**Comment by matthewdupre** [comment]
Squash please, then good :).

**Comment by matthewdupre** [comment]
LGTM

**Pull request merged**